HoloPipeLines-Models/dense_vnet_abdominal_ct/README.md

```diff
@@ -0,0 +1,73 @@
+# Niftynet abdominal neural network container
+This page describes how to create a docker image that is with the endpoint for the network described in
+
+Eli Gibson, Francesco Giganti, Yipeng Hu, Ester Bonmati, Steve Bandula, Kurinchi Gurusamy, Brian Davidson, Stephen P. Pereira, Matthew J. Clarkson and Dean C. Barratt (2017), Automatic multi-organ segmentation on abdominal CT with dense v-networks https://doi.org/10.1109/TMI.2018.2806309
+
+This network model container segments eight organs on abdominal CT, comprising the gastointestinal tract (esophagus, stomach, duodenum), the pancreas, and nearby organs (liver, gallbladder, spleen, left kidney).
+
+Here is the repo for the network model
+https://github.com/NifTK/NiftyNetModelZoo/tree/5-reorganising-with-lfs/dense_vnet_abdominal_ct
+the repo only runs in the niftynet framework so to run the network model locally you need to follow the isntruction from the niftynet and download the framework first
+https://niftynet.readthedocs.io/en/dev/installation.html
+
+
+## Build Docker image from the Docker file
+
+```
+docker build -t [docker name:tag] .
+# example
+docker build -t niftynet:latest .
```

Comment:
Is it possible to reference a specific tag instead of latest? The concern here is that latest is ever-changing, which may break your overall system.

Reply:
Sorry, just realised that this is not really an issue as you are just building the container from your own dockerfile.